lib/resolvers/establishment.js

```diff
@@ -1,7 +1,29 @@
 const resolver = require('./base-resolver');
-const { pick } = require('lodash');
+const { pick, omit } = require('lodash');
 
 module.exports = ({ models }) => ({ action, data, id }, transaction) => {
+  const { Establishment, Authorisation } = models;
+
+  if (action === 'update') {
+    return Establishment.query(transaction)
+      .patchAndFetchById(id, data)
```

Comment:
Does the fact that `data.authorisations` exists not cause an issue here, or are the extra properties silently ignored?